scripts/docker_container_reboot.py

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/python3
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+environment = os.environ.get('environment')
+
+
+def restartContainers():
+    composeDown = 'docker-compose -f docker-compose.prod.yml down'
+    composeUp = 'docker-compose -f docker-compose.prod.yml up -d'
+
+    if "atlas" in environment.lower():
+        os.system('cd ../atlas/repository && ' + composeDown)
```

Comment:
The KE and Atlas front-end are running on different machines, so I think you'll need to make the environment check more specific and only run the "docker-compose up" on the appropriate machine.

Reply:
Good call, I forgot about that split, can make that update